packages/scripts/config/webpack.config.js

```diff
@@ -67,7 +67,7 @@ const config = {
 		// WP_LIVE_RELOAD_PORT global variable changes port on which live reload works
 		// when running watch mode.
 		! isProduction && new LiveReloadPlugin( { port: process.env.WP_LIVE_RELOAD_PORT || 35729 } ),
-		new DependencyExtractionWebpackPlugin(),
+		new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ),
```

Comment:
Includes `wp-polyfill` as a script dependency for every entrypoint.

Reply:
> Includes `wp-polyfill` as a script dependency for every entrypoint.

Looking at the implementation, would `wp-polyfill` include `wp-polyfill` as a dependency of itself?

Reply:
It would yes.

That said, I don't see polyfill being built as part of Gutenberg (no `build/polyfill/index.deps.json`). Is this a theoretical concern or is the polyfill built somewhere we'd need to be careful with this?

Reply:
I don't think we will ever scan `wp-polyfill` but we can add guard clause to ba on the safe side.